compose.py

```diff
@@ -73,7 +73,7 @@ def start_dockerfile(ddir, images, argmap, dockerfile_name, backends):
 
     import build
     df += build.dockerfile_prepare_container_linux(argmap, backends,
-                                                   FLAGS.enable_gpu,
+                                                   not FLAGS.disable_gpu,
```

Comment:
We might want to rename the variables and re-work the checks for disable_gpu instead of using enable_gpu